spec/javascripts/health-care/components/personal-information/VeteranAddressSection.spec.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+
+import VeteranAddressSection from '../../../../../_health-care/_js/components/personal-information/VeteranAddressSection';
+
+describe('<VeteranAddressSection>', () => {
+  const mockAddress = [{
```

Comment:
or `nullAddress` is probably more descriptive.

Naming something after a pattern is usually the last type of name I go for since it's largely non-descriptive...